src/app/src/main/kotlin/org/imperial/mrc/hint/clients/FlowClient.kt

```diff
@@ -0,0 +1,27 @@
+package org.imperial.mrc.hint.clients
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import org.imperial.mrc.hint.models.ErrorReport
+import org.springframework.http.ResponseEntity
+import org.springframework.stereotype.Component
+
+interface Flow
+{
+    fun notifyTeams(url: String, data: ErrorReport): ResponseEntity<String>
+}
+
+@Component
+class FlowClient(val objectMapper: ObjectMapper) : FuelClient(), Flow
```

Comment:
As I mentioned in my previous review, and implied by Alex's suggested code, the FlowClient should configure itself:
```suggestion
class FuelFlowClient(val objectMapper: ObjectMapper) : FuelClient(appProperties.issueReportUrl), FlowClient
```
This way the Controller doesn't need to concern itself with the relevant application property (which is an implementation detail of the Flow client).

Also notice use of `FuelFlowClient` i.e. a Fuel-backed implementation of `FlowClient`.